posthog/models/team/team.py

```diff
@@ -217,6 +217,26 @@ def _person_on_events_querying_enabled(self) -> bool:
         # on self-hosted, use the instance setting
         return get_instance_setting("PERSON_ON_EVENTS_ENABLED")
 
+    @property
+    def recordings_list_v2_query_enabled(self) -> bool:
```

Comment:
Could we do this on the frontend instead and control it via a query param to the API? Makes it much easier to test back and forth in a live situation to compare results etc.

Reply:
Hmm, on second thought, I don't want to add filters to the object and muddy our caching for insights. You think it's worth it for this?

Reply:
I wasn't suggesting adding it to filters (not everything has to go in that mega object 😅 ) - just a standard query param. I pushed the change in the way I meant so now we just check a query param which makes it super easy to flip back and forth without having to change the feature flag all the time